tests/storage/storage.go

```diff
@@ -646,21 +649,21 @@ var _ = SIGDescribe("Storage", func() {
 
 			// Not a candidate for testing on NFS because the VMI is restarted and NFS PVC can't be re-used
 			It("[test_id:3138]should start vmi multiple times", func() {
-				vmi = tests.NewRandomVMIWithPVC(tests.DiskAlpineHostPath)
-				tests.AddPVCDisk(vmi, "disk1", v1.DiskBusVirtio, tests.DiskCustomHostPath)
+				vmiSpec := libvmi.NewAlpine(libvmi.WithPersistentVolumeClaim(tests.DiskAlpineHostPath))
+				vmiSpec = tests.AddPVCDisk(vmiSpec, "disk1", v1.DiskBusVirtio, tests.DiskCustomHostPath)
 
 				num := 3
 				By("Starting and stopping the VirtualMachineInstance number of times")
 				for i := 1; i <= num; i++ {
-					obj := tests.RunVMIAndExpectLaunch(vmi, 240)
+					obj := tests.RunVMIAndExpectLaunch(vmiSpec, 240)
 
 					// Verify console on last iteration to verify the VirtualMachineInstance is still booting properly
 					// after being restarted multiple times
 					if i == num {
 						By("Checking that the second disk is present")
-						Expect(console.LoginToAlpine(vmi)).To(Succeed())
+						Expect(console.LoginToAlpine(obj)).To(Succeed())
```

Comment:
Seems like an unrelated fix (fits a different commit or PR). But I can look the other way :)

Reply:
You are right, I should do it in https://github.com/kubevirt/kubevirt/pull/7634.